test/unit/test.ts

```diff
@@ -0,0 +1,34 @@
+/*
+	File: test/unit/unit-test.ts
+	cpuabuse.com
+*/
+
+/**
+ * Performs a series of unit tests.
+ */
+
+import { testNumber } from "./pipeline/number";

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "./pipeline/number"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5058624)